.github/workflows/hulk_deployment.yaml

```
@@ -18,6 +18,7 @@ jobs:
       - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)